features/mobile/not_safe_for_work.feature

```diff
@@ -0,0 +1,74 @@
+@javascript @mobile
+Feature: Not safe for work
+  Background:
+    Given following users exist:
+      | username    | email             |
+      | pr0n king   | tommy@pr0n.xxx    |
```

Comment:
@AugierLe42e I agree with @denschub. Please change the domains to example.com while rebasing your PR.